### PR TITLE
New version of interactivetool_metashark.xml who deal with GIS files

### PR DESCRIPTION
**Pull request by yvanlebras**
(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the changes. The "why" may simply be citing a relevant Galaxy issue.)
(If fixing a bug, please add any relevant error or traceback)
(For UI components, it is recommended to include screenshots or screencasts)

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: tools/interactive/interactivetool_metashark.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<tool id="interactive_tool_metashark" tool_type="interactive" name="metashark" version="1.7.3" profile="22.01">
+<tool id="interactive_tool_metashark" tool_type="interactive" name="metashark" version="1.7.35" profile="22.01">
     <description>Metadata Shiny Automated Resource and Knowledge</description>
     <requirements>
-        <container type="docker">ylebras/metashark:1.7.3</container>
+        <container type="docker">ylebras/ms_annot_gis_patnat_dev_072023</container>
```
Comment (bgruening) [inline tools/interactive/interactivetool_metashark.xml]: Is that really the container name? No version number?

Comment (yvanlebras) [reply tools/interactive/interactivetool_metashark.xml]: This is the image name yes. We will have a total new version normally at the end of the year

Comment (yvanlebras) [reply tools/interactive/interactivetool_metashark.xml]: true... maybe it is better i tag this image something like metashark:1.7.35 ?

Comment (bgruening) [reply tools/interactive/interactivetool_metashark.xml]: yes, that would be cleaner

Comment (yvanlebras) [reply tools/interactive/interactivetool_metashark.xml]: Done! Hope I don't make any mistake! THANK YOU!!!

**Review by bgruening — APPROVED** [review]

**Pull request merged**